<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BinColorsDialogBase</class>
 <widget class="QDialog" name="BinColorsDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Colors</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>103</height>
      </size>
     </property>
     <property name="title">
      <string>Mode</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButtonDefault">
        <property name="text">
         <string>Default</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>246</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButtonCustom">
        <property name="text">
         <string>Custom</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonFillDefaultColors">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Fill bin list with default colors</string>
        </property>
        <property name="text">
         <string>Fill using default colors</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>380</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" rowspan="3">
    <widget class="QTabWidget" name="tabWidgetBins">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSoftBin">
      <attribute name="title">
       <string>Software Bin</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="2">
        <widget class="Line" name="Line1">
         <property name="toolTip">
          <string>Give a name to your dataset query</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::VLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="buttonRemoveSoftBin">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>29</horstretch>
           <verstretch>25</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>29</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>29</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Delete a Soft-Bin entry</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="gex.qrc">
           <normaloff>:/gex/icons/file_remove.png</normaloff>:/gex/icons/file_remove.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="buttonSoftBinProperties">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>29</horstretch>
           <verstretch>25</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>29</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>29</width>
           <height>25</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Edit / Review a Soft-Bin enrty</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="gex.qrc">
           <normaloff>:/gex/icons/properties.png</normaloff>:/gex/icons/properties.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="5">
        <widget class="QListWidget" name="ListWidgetSoftBin"/>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="buttonNewSoftBin">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>29</horstretch>
           <verstretch>25</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>29</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>29</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Add a new Soft-Bin entry</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="gex.qrc">
           <normaloff>:/gex/icons/group.png</normaloff>:/gex/icons/group.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabHardBin">
      <attribute name="title">
       <string>Hardware Bin</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="buttonNewHardBin">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>29</horstretch>
           <verstretch>25</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>29</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>29</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Add a new Hard-Bin entry</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="gex.qrc">
           <normaloff>:/gex/icons/group.png</normaloff>:/gex/icons/group.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="buttonHardBinProperties">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>29</horstretch>
           <verstretch>25</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>29</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>29</width>
           <height>25</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Edit / Review a Hard-Bin enrty</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="gex.qrc">
           <normaloff>:/gex/icons/properties.png</normaloff>:/gex/icons/properties.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="Line" name="Line1_2">
         <property name="toolTip">
          <string>Give a name to your dataset query</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::VLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="buttonRemoveHardBin">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>29</horstretch>
           <verstretch>25</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>29</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>29</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Delete a Hard-Bin entry</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="gex.qrc">
           <normaloff>:/gex/icons/file_remove.png</normaloff>:/gex/icons/file_remove.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>208</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="5">
        <widget class="QListWidget" name="ListWidgetHardBin"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSite">
      <attribute name="title">
       <string>Site</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QPushButton" name="buttonNewSite">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>29</horstretch>
           <verstretch>25</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>29</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>29</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Add a new Site entry</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="gex.qrc">
           <normaloff>:/gex/icons/group.png</normaloff>:/gex/icons/group.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="buttonSiteProperties">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>29</horstretch>
           <verstretch>25</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>29</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>29</width>
           <height>25</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Edit / Review a Site enrty&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="gex.qrc">
           <normaloff>:/gex/icons/properties.png</normaloff>:/gex/icons/properties.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="Line" name="Line1_3">
         <property name="toolTip">
          <string>Give a name to your dataset query</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::VLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="buttonRemoveSite">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>29</horstretch>
           <verstretch>25</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>29</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>29</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Delete a Site entry</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="gex.qrc">
           <normaloff>:/gex/icons/file_remove.png</normaloff>:/gex/icons/file_remove.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>223</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="5">
        <widget class="QListWidget" name="ListWidgetSite"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="PushButtonOk">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>67</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="PushButtonCancel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>67</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gex.qrc"/>
 </resources>
 <connections/>
</ui>
